adoptopenjdk-api-v3-updater/src/main/kotlin/net/adoptopenjdk/api/v3/stats/DockerStatsInterface.kt

```diff
@@ -40,15 +41,21 @@ class DockerStatsInterface {
 
         return pullAllStats()
                 .map {
-                    DockerDownloadStatsDbEntry(now, it.getJsonNumber("pull_count").longValue(), it.getString("name"))
+                    DockerDownloadStatsDbEntry(
+                        now,
+                        it.getJsonNumber("pull_count").longValue(),
+                        it.getString("name"),
+                        "openjdk[0-9]+".toRegex().find(it.getString("name"))!!.value.substring(7).toInt(),
```

Comment:
fairly complex line, could probably extract it into a method, that 7 also seems kind of brittle, can use a named capture group:
```
val x = "openjdk(?<featureNum>[0-9]+)".toRegex().matchEntire("openjdk99")?.groups?.get("featureNum")?.value?.toInt()
```